.changelog/unreleased/improvements/857-make-handshake-cancelable.md

```diff
@@ -0,0 +1 @@
+- `[node]` Make handshake cancelable. ([cometbft/cometbft\#857](https://github.com/cometbft/cometbft/pull/857))
```

Comment:
```suggestion
- `[node]` Make handshake cancelable ([cometbft/cometbft\#857](https://github.com/cometbft/cometbft/pull/857))
```

Reply:
Please add two separate entries under the "breaking" section to indicate that the interfaces of the `Handshaker.Handshake` and `NewNode` methods now require contexts?